packages/core/src/updateContext.ts

```diff
@@ -29,6 +29,12 @@ export function updateContext<TContext, TEvent extends EventObject>(
   }
   const capturedActions: InvokeActionObject[] = [];
 
+  if (!context) {
```

Comment:
dont we already warn about this 4 LOCs above?

Reply:
Yes, we can remove the warning.